packages/combobox/src/Combobox.spec.tsx

```diff
@@ -1140,14 +1140,14 @@ describe('packages/combobox', () => {
       });
 
       describe('Right arrow key', () => {
-        test('Does nothing when focus is on clear button', () => {
+        test('Focuses on inputEl when on clear button', () => {
```

Comment:
I'm not sure about this one. If we do want the focus to cycle around, should it always focus on the input, or on the first chip?

Reply:
Also, if this is the behavior we want, pressing left from the first chip in MS should focus the clear button. (and, potentially, `left` from the beginning of the text in SS)

Reply:
Resetting this test, bc per above no longer manually setting focus back to input onKeydown